linode/manager
# upcoming: [M3-7609] - Placement Groups Landing Page

## Description 📝
This PR adds core elements for the Placement Group feature, namely basic navigation and a populated landing page

## Changes  🔄
- Adds PrimayNav entry w/ Beta Chip
- Adds a populated landing page with table and rows
- Tests for components

## Preview 📷
![Screenshot 2024-01-18 at 14 17 27](https://github.com/linode/manager/assets/130582365/d49987c5-4c69-48c1-b5ad-42492818ba3f)



## How to test 🧪

### Prerequisites

### Verification steps 
- Pull and run code locally
- in Dev settings, enable both the `Placement Groups` flag and MSW
- Click on "Placement Groups" in the menu
- Confirm the landing page content

## As an Author I have considered 🤔

*Check all that apply*
